### PR TITLE
Book Detail IV: Badge

### PR DESCRIPTION
**Pull request by mcolombo87**
## Summary

New badge over book cover.

## Screenshots

image

## Trello Card

https://trello.com/c/uq3CXo18/29-detalle-de-libro-iv-badge

### CODE REVIEW

File: maquetado/scss/details.scss [hunk 1]
```diff
@@ -59,4 +51,25 @@
...
   }
+
+  .book-cover {
+    position: relative;
+
+    .book-image {
+      height: 100%;
+      max-height: 416px;
+      max-width: 309px;
+      padding: 24px;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Is it necessary this padding?

Comment (mcolombo87) [reply maquetado/scss/details.scss]: It's not. I change that padding (conceptually wrong) and I add that padding in your container. Another padding was eliminated and add margin between brothers.

File: maquetado/scss/details.scss [hunk 2]
```diff
@@ -59,4 +51,25 @@
...
+
+    .book-image {
+      height: 100%;
+      max-height: 416px;
+      max-width: 309px;
+      width: 100%;
+    }
+
+    .book-badge {
+      max-height: 118.57px;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Change to 118px or 119px

File: maquetado/scss/details.scss [hunk 3]
```diff
@@ -59,4 +51,25 @@
...
+      max-width: 94.53px;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Same here

File: maquetado/scss/details.scss (hunk 2 above)
Comment (mcolombo87) [reply maquetado/scss/details.scss]: done

File: maquetado/scss/details.scss (hunk 3 above)
Comment (mcolombo87) [reply maquetado/scss/details.scss]: done

**Pull request merged**